### ISSUE TITLE
[Adaptive Cards- Web Render Test on Teams]: Voice Over is conveying label information with all options while navigating on the radio buttons present on the screen.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#410]
### Target Platforms

iOS

### SDK Version

7.13

### Application Name

Microsoft Teams

### Problem Description

**Test Environment:**
Device: iPhone 12
iOS Version: 18.5
Teams Application Version: 7.13
Screen Reader: Voice Over

**Pre-Requisite:**
Turn on Voice Over.

**Repro Steps:**
1. Open the application using double tap.
2. Perform right swipe, navigate to the AC Web render group, and double tap to enter in the group.
3. Navigate to any poll using swipe gesture.
4. Now perform the right swipe and navigate to the radio buttons present on the card and observe the issue.

**Actual Result:**
While navigating on the radio buttons present on the 'What is your favourite color?' Voice over is announcing label information with all radio buttons. Also, voice over does not announce the position with all radio buttons. Here, it is announcing as **'What is your favourite color question mark comma red comma radio button double tap to select.'**

**Expected Result:**
Screen Reader should not announce label information with all radio buttons. While navigating in forward navigation, It should announce label info with first label and while navigating in reverse navigation it should announce label information with last label.
Also it should announce the position of the radio buttons. It should announce as **'What is your favourite color question mark red radio button 1 of 5 not selected double tap to select.'**

**Note:**
Same issue is repro throughout the screen.
After selecting the radio button screen reader is also announcing label information.

**User Impact:**
It will impact the screen reader user as they will get irritate if the label information is announced for all radio buttons also, they will not be able to know the number of radio buttons present on the screen.

**WCAG Reference:**
[Understanding Success Criterion 1.3.1: Info and Relationships | WAI | W3C](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)

Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

https://github.com/user-attachments/assets/55addaf0-b2c6-4a25-a1ea-5504287d5bc9

### Card JSON

```JSON
NA
```
